highfidelity/hifi
# Entity list - reorder columns

Allow columns in the entity list to be swapped by clicking and dragging header cells.

Addresses: https://highfidelity.fogbugz.com/f/cases/19845

Pre-Merge Test Plan:
- Enter Interface in desktop into any domain with edit permissions and with many different entities
- Open Create to the Entity List
- Verify if clicking down on column header cells and releasing without moving the mouse that the column sort toggle is triggered (same as before this PR)
- Verify if clicking on the lines between column header cells that the columns can be resized (same as before this PR)
- Verify if clicking down on column header cells and dragging the header cell left and right that it will swap columns with it's neighbor (verify all row cells swap as expected in addition to the header cell) and it will display the header cell in a light blue tint until released
- Add additional columns using the dropdown above the scrollbar
- Stress test moving columns around as well as more sorting and resizing
